src/TreeMessageUtils.jl

```diff
@@ -574,7 +574,7 @@ during clique inference.
 
 DevNotes
 - TODO make `::Vector{Symbol}` version.
-
+- TODO function taking fcts::Vector{DFGFactor} is unused and replace by the tags version, perhaps we can remove it. 
```

Comment:
just making sure, `tags` or `labels`?

Reply:
tags, If you remember we remove messages based on the `__LIKELIHOOD_MESSAGE__` tag